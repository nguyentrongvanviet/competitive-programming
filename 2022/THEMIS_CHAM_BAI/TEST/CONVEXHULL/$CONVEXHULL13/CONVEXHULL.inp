38
128803346 -49055749
221650808 -193757121
685496490 650572746
101752208 21888123
51141510 122017984
235261588 829957070
-1473492 14663437
-229471952 -202232349
90305089 -233194419
-244492605 782577390
757710132 -584495863
92085448 720690973
154489740 111618699
161779369 909292036
-370277794 -344525524
-118370993 17996276
517740368 451550456
-713931631 360579831
383822869 -60751809
-408306046 -97238105
-510694217 238419917
-249003715 -561004506
493737443 -388758464
212772834 205808281
-867971840 86995547
164758521 680811348
383689972 -219872403
64437938 19577953
-426497448 -494572279
-198205486 290185976
381799806 -774050499
393489976 -619802560
430074187 -714823882
-3114912 -62674837
551209880 -73538577
330460773 -16033124
-118406867 120638985
669815016 -118582979
