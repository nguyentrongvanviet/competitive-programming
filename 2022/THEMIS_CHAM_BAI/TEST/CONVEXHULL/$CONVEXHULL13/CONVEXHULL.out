9
1917393925731780073.0
381799806 -774050499
757710132 -584495863
685496490 650572746
161779369 909292036
-244492605 782577390
-713931631 360579831
-867971840 86995547
-426497448 -494572279
-249003715 -561004506
